istioctl/cmd/istiodconfig.go

```diff
@@ -471,12 +474,11 @@ func istiodLogCmd() *cobra.Command {
 func istiodConfig() *cobra.Command {
 
 	istiodConfigCmd := &cobra.Command{
-		Use:   "istiod-config",
+		Use:   "istiod",
 		Short: "Manage control plane (istiod) configuration",
 		Long:  "A group of commands used to manage istiod configuration",
-		Example: ` # Retrieve information about istiod configuration.
-  istioctl istiod-config <log>`,
-		Aliases: []string{"idc"},
+		Example: `  # Retrieve information about istiod configuration.
+  istioctl istiod-config log`,
```

Comment:
Is there 2 different command are there is only one?
- istioctl x istiod log
- istioctl x istiod-config log

Reply:
Only one.